src/main/kotlin/com/gamingbacklog/api/gamingbacklogapi/models/requests/UpdateLibraryGamesRequest.kt

```diff
@@ -0,0 +1,6 @@
+package com.gamingbacklog.api.gamingbacklogapi.models.requests
+
+class UpdateLibraryGamesRequest(
+  var gameId: String
```

Comment:
Question (I'm not actually sure about REST API best practices) - why are we including this as a request object and not a path param?

Reply:
I've seen it done both ways, but I believe the best practice for POST/updates like this is to send it via a response body rather than a param.